docs/current/data-model-howto.md

```diff
@@ -17,7 +17,13 @@
 </head><article class="markdown-body">
 
 # Vaquero Data Model and YOU
+[Home](https://ciscocloud.github.io/vaquero-docs/)
 
+- [Example Data Models](https://github.com/gem-test/vaquero)
+
+## Data Model Diagram to show relationships
+Note re-use of `host groups`, `os` and `assets`. Each site contains an `environment file` and an `inventory file` that lists all related hosts. Hosts are members of a single `host group`. `Host groups` reference `os` and `assets`
+![](https://raw.githubusercontent.com/CiscoCloud/vaquero-docs/docs-cleanup/docs/current/dm-picture.png)
```

Comment:
per Drakes request, if we want a more formal. ER diagram I could do that too. out of laziness of having to refresh myself on ER and how ugly it would be. I made the personal choice that it wasn't worth it